testutil/staking-rewards.go

```diff
@@ -114,10 +114,14 @@ func PrepareAccountsForDelegationRewards(t *testing.T, ctx sdk.Context, app *app
 
 		// TODO: Replace this with testutil.Commit? Will only be possible after test suite setup cleanup
 		// because some suites use an initial height != 1, which is not accounted for, so there's a mismatch of expected vs. actual block height
+		ctx, err := Commit(ctx, app, time.Second*2, nil)
+		if err != nil {
+			return sdk.Context{}, err
+		}
 
-		// end block to bond validator and increase block height
-		staking.EndBlocker(ctx, app.StakingKeeper)
-		ctx = ctx.WithBlockHeight(ctx.BlockHeight() + 1)
+		//// end block to bond validator and increase block height
+		//staking.EndBlocker(ctx, app.StakingKeeper)
+		//ctx = ctx.WithBlockHeight(ctx.BlockHeight() + 1)
```

Comment:
this change (using `Commit` instead of `staking.EndBlocker`) breaks tests that use `EthSetup` from [eth_helpers.go](https://github.com/evmos/evmos/blob/c7ce2fc13d6089456c463fbd70c81ab7e1dbc923/app/ethtest_helper.go) (e.g. [here](https://github.com/evmos/evmos/blob/c7ce2fc13d/app/ante/evm/setup_test.go#L45-L72)) and start the chain e.g. [from height 2](https://github.com/evmos/evmos/blob/c7ce2fc13d6089456c463fbd70c81ab7e1dbc923/app/ante/evm/setup_test.go#L74). This would have to be aligned in the [call to `InitChain`](https://github.com/evmos/evmos/blob/c7ce2fc13d6089456c463fbd70c81ab7e1dbc923/app/ethtest_helper.go#L89-L96).